### ISSUE TITLE
Change click me to a suitable label for each link

### ISSUE THREAD

**Issue by emmalearnscode** [goobric/linkfreeInformation#1]
Using the words click me for every link on a page is not accessible for anyone using a screen reader, all of them will be read out as 'link, click me' 

Suggestion:
Instead of this: `<p>What is NPM, Tutorial for beginners <a href="https://www.youtube.com/watch?v=P3aKRdUyr0s" target="_blank">click here</a></p>`

Just use: `<a href="https://www.youtube.com/watch?v=P3aKRdUyr0s" target="_blank">What is NPM, a tutorial for beginners</a>`

**Comment by goobric** [comment]
TY sm for your input and advice Emma, yes for accessibility this is very important.
I have updated the markup
